### ISSUE TITLE
Switch to karma?

### ISSUE THREAD

**Issue by kahwee** [kahwee/range-multiple#1]
@thomato, you mentioned that you may need to test for mouse events. Shall we switch to Karma or something? I think it would be nice if we can easily see how the test cases run. 

Do you have suggestions for other tools or we use `karma` for this?

**Comment by engwansong** [comment]
@kahwee Yes, I think we should switch to Karma. The only reason I opted for jsdom was to get started. One thing to consider is the ability to run tests remotely, but I think something like karma-phantomjs-launcher will work for that.
